trade-finance-manager-ui/server/controllers/case/index.test.js

```diff
@@ -99,12 +99,12 @@ describe('controllers - case', () => {
       it('should check whether deal cancellation is enabled', async () => {
         await caseController.getCaseDeal(req, res);
 
-        expect(isDealCancellationEnabled).toHaveBeenCalledTimes(1);
+        expect(canDealBeCancelled).toHaveBeenCalledTimes(1);
```

Comment:
Should we add `calledWith()` assertions?

Reply:
Have added, thank you!